config/settings/development.yml

```diff
@@ -2,7 +2,7 @@ codeocean:
   url: http://localhost:7000
 omniauth:
   samltestid:
-    enable: true
+    enable: false
```

Comment:
This change seems to be unrelated, see #1281.

Reply:
I accidentally committed that.